NOLF2/ClientShellDLL/ClientShellShared/ClientFXDB.cpp

```diff
@@ -217,8 +217,6 @@ void CClientFXDB::Term()
 		pGroupNode = pGroupNode->m_pNext;
 	}
 	m_collGroupFX.RemoveAll();
-
-	UnloadFxDll();
```

Comment:
instread of removing it from here, fix it that CClientFXDB::Term() can safely be called multiple times

(like by code, and by the destructor when main exits later)

Reply:
if you remove it from here... you will have to call it in the destructor
of the DLL isn't cleanly unloaded when ClientFXDB is  destroyed